app/src/main/java/com/kabouzeid/gramophone/model/lyrics/Lyrics.java

```diff
@@ -0,0 +1,49 @@
+package com.kabouzeid.gramophone.model.lyrics;
+
+import com.kabouzeid.gramophone.model.Song;
+
+public class Lyrics {
+    public Song song;
+    public String data;
+
+    boolean parsed = false;
+    boolean valid = false;
+
+    public Lyrics(Song song, String data) {
+        this.song = song;
+        this.data = data;
+    }
+
+    public static Lyrics parse(Song song, String data) {
```

Comment:
Do you have any idea how to iterate through all available formats and fallback on plain `Lyrics` without hardcoding them in one method? Maybe something like static ArrayList of available parsers?